cluster/examples/kubernetes/ceph/crds.yaml

```diff
@@ -1,415 +1,254 @@
-                        iteration:
-                          type: integer
-                          format: int32
-                security:
-                  type: object
-                  properties:
-                    kms:
-                      type: object
-                      properties:
-                        connectionDetails:
-                          type: object
-                          nullable: true
-                          x-kubernetes-preserve-unknown-fields: true
-                        tokenSecretName:
-                          type: string
-                logCollector:
-                  type: object
-                  properties:
-                    enabled:
-                      type: boolean
-                    periodicity:
-                      type: string
-                annotations:
-                  type: object
-                  nullable: true
-                  x-kubernetes-preserve-unknown-fields: true
-                placement:
-                  type: object
-                  nullable: true
-                  x-kubernetes-preserve-unknown-fields: true
-                labels:
-                  type: object
-                  nullable: true
-                  x-kubernetes-preserve-unknown-fields: true
-                resources:
-                  type: object
-                  nullable: true
-                  x-kubernetes-preserve-unknown-fields: true
-                healthCheck:
-                  type: object
-                  nullable: true
-                  x-kubernetes-preserve-unknown-fields: true
-                priorityClassNames:
-                  type: object
-                  nullable: true
-                  x-kubernetes-preserve-unknown-fields: true
-            status:
-              type: object
-              x-kubernetes-preserve-unknown-fields: true
-      additionalPrinterColumns:
-        - name: DataDirHostPath
-          type: string
-          description: Directory used on the K8s nodes
-          jsonPath: .spec.dataDirHostPath
-        - name: MonCount
-          type: string
-          description: Number of MONs
-          jsonPath: .spec.mon.count
-        - name: Age
-          type: date
-          jsonPath: .metadata.creationTimestamp
-        - name: Phase
-          type: string
-          description: Phase
-          jsonPath: .status.phase
-        - name: Message
-          type: string
-          description: Message
-          jsonPath: .status.message
-        - name: Health
-          type: string
-          description: Ceph Health
-          jsonPath: .status.ceph.health
-      subresources:
-        status: {}
-# OLM: END CEPH CRD
-# OLM: BEGIN CEPH CLIENT CRD
+                      timeout:
+                        type: string
+                    type: object
+                type: object
+                x-kubernetes-preserve-unknown-fields: true
+            required:
+            - compressionMode
```

Comment:
Is this actually required?

Also of note: I don't see this in any of our docs.